src/Umbraco.Infrastructure/Search/ExamineComponent.cs

```diff
@@ -515,6 +515,7 @@ private void ReIndexForContent(IContent sender, bool isPublished)
                 DeferedReIndexForContent.Execute(this, sender, isPublished);
         }
 
+        //TODO: this causes an exception when deleting a member
```

Comment:
@Shazwazza I'm finding that when I break on all exceptions after deleting a member, I come to this point (line 258) in the `ExamineComponent.cs`, where the member has already been deleted but it tries to reindex it. 

![image](https://user-images.githubusercontent.com/1649855/104222159-ccc0f980-5439-11eb-9869-000257e91148.png)

The member service is returning a null member object even though it hasn't matched anything against the Payload (populated with the now deleted member details)

![image](https://user-images.githubusercontent.com/1649855/104222349-127dc200-543a-11eb-8abc-c5493521f8a8.png)

This means it tries to run the `ReindexForMember` method but fails because the properties are null. 

Is this something you have seen before or something new introduced by this update do you think?

Reply:
![image](https://user-images.githubusercontent.com/1649855/104476888-a1184d80-55b8-11eb-800a-7f3d5dfa30b9.png)

Just checked in v8 and same behaviour happens when deleting a member. `DeferedReIndexForMember` is called and then the member properties are null, an exception happens under the hood. So this isn't a recently introduced issue. Should I create a ticket for it in v8?

Reply:
Closing as created related issue #9684